ltopers.rb

```diff
@@ -1,9 +1,9 @@
 class Ltopers < Formula
   desc "Scripts for doing stuff with LTFS"
   homepage "https://github.com/amiaopensource/ltopers"
-  url "https://github.com/amiaopensource/ltopers/archive/v2020-01-27.tar.gz"
-  version "2020-01-27"
-  sha256 "7d3ba39bab196e352070ce5a4457113bf8682cdbc54469bc16f8f498313781bb"
+  url "https://github.com/amiaopensource/ltopers/archive/refs/tags/v20230823.tar.gz"
```

Comment:
Is the change to `/archive/refs/tags/` from just `/archive/` in the HTML more robust?

Reply:
Thanks! No, I don’t think it’s more robust. I’ll change this.